peregrine/blueprints/datasets.py

```diff
@@ -60,3 +61,33 @@ def get_datasets():
         node = match.group(2)
         result[projects[index]][node] = value
     return flask.jsonify(result)
+
+
+@blueprint.route("/projects", methods=["GET"])
+def get_projects():
+    """
+    Get all projects high level information, if a deployment is configured
+    to set PUBLIC_DATASETS to True, this endpoint will be open to
+    anonymous users
+    """
+    if os.environ.get("PUBLIC_DATASETS", "false").lower() == "true":
+        set_read_access_projects_for_public_endpoint()
+    else:
+        set_read_access_projects()
+    projects = flask.g.read_access_projects
+    if not projects:
+        return flask.jsonify({})
```

Comment:
Should be either empty array `[]` or `{projects: []}` depending on your answer to the comment below